src/easyzoom.js

```diff
@@ -238,6 +246,7 @@
      */
     EasyZoom.prototype.hide = function() {
         if (!this.isOpen) return;
+        if (!this.opts.beforeHide.call(this)) return;
```

Comment:
Same as above. Perhaps the default value for the callbacks should be `null` so we can test whether or not to call the function?